chart/demo-app/templates/github-secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: github-token
+type: Opaque
+data:
+  GITHUB_TOKEN: dG9wU2VjcmV0MTIz
```

Comment:
Can you please move this to the README with instructions how to create this TOKEN and then the kubectl command to create or apply it?